infra/modules/service/variables.tf

```
@@ -89,3 +89,8 @@ variable "extra_policies" {
   type        = map(string)
   default     = {}
 }
+
+variable "is_temporary" {
+  description = "Boolean value to determine if this resource should disable deletion protection. Needed for automated test cleanup"
```

nit: the type already says it's a boolean so a bit redundant to say that in the description

```suggestion
  description = "Whether the service is meant to be spun up temporarily (e.g. for automated infra tests). This is used to disable deletion protection for the load balancer."
```